invesalius/data/viewer_volume.py

```diff
@@ -876,7 +876,8 @@ def CreateTargetGuide(self):
             for ind in self.guide_arrow_actors:
                 self.target_guide_renderer.RemoveActor(ind)
 
-        obj_polydata = vtku.CreateObjectPolyData(self.coil_path)
+        coil_path = os.path.join(inv_paths.OBJ_DIR, "magstim_fig8_coil.stl")
```

Comment:
Write one comment before this line, explaining the choice and what could be considered in the future. Customized target coil models. Now we use only a default Fig-8